ztag/schema.py

```diff
@@ -563,7 +563,7 @@ class CensysString(WhitespaceAnalyzedString):
 #    "metadata":local_metadata
 #})
 
-ztag_xssh_signature = SubRecord({
+ztag_ssh_signature = SubRecord({
```

Comment:
This name was the only place that a reference to the new **X**SSH grabber leaked through. It seems cleaner for ZTag's schema to refer to this simply as "SSH", since aside from the transform implementation ZTag has no awareness of XSSH.